cran-comments.md

```diff
@@ -1,7 +1,8 @@
 # Update - v0.5.1
-- functionality to detect and remove locally cached data that has been recalled
-- automatic checks that warn if locally cached data has been recalled
-- rename get/set cache/api_key functions to avoid name conflicts with other packages
+- Added functionality to detect and remove locally cached data that has been recalled by Statistics Canada
+- added automatic checks that warn if locally cached data has been recalled
```

Comment:
concerned this might be an issue

Reply:
I think the automatic checks are good, users don't read the package notes or re-read the README/documentation when updating packages. Warning users that they are using recalled data is important.

Reply:
Yeah, just needed to rephrase that this is part of a user's initial call and is activated by that only, not something that happens in the background without the user knowing